JotunnLib/Managers/PrefabManager.cs

```diff
@@ -441,7 +441,7 @@ public static Object GetPrefab(Type type, string name)
                         // load an never release reference to keep it loaded
                         asset.Load();
 
-                        if (asset.Asset.GetType() == type)
+                        if (asset.Asset != null && asset.Asset.GetType() == type)
```

Comment:
If a null check is added here, the statement below has to be covered too. This rather points to Reforged adding an asset that doesn't live anymore but a check definitely won't hurt. Although it's unexpected since asset.IsValid returns true, best to ckeck how Reforged is adding assets into the vanilla sytem.

`asset.Asset &&` should rather be used than `asset.Asset != null &&`, because the first one is checking for Unity liveness and is more reliable and much cheaper.

Reply:
Makes sense.  I'll make some adjustments.